examples/elk/README.md

```diff
@@ -0,0 +1,46 @@
+# converge-elk
+
+A Converge example that sets up a single node docker-based [ELK](https://www.elastic.co/webinars/introduction-elk-stack) stack.
+
+[Filebeat](https://www.elastic.co/products/beats/filebeat) is used instead of [Logstash](https://www.elastic.co/products/logstash) for the log collection component.
+
+## Usage
+
+### Vagrant
+
+In the [Vagrantfile](./Vagrantfile), change the file provisioner source to point to a version of the converge binary built with the `linux/amd64` OS architecture.
```

Comment:
Better yet, could the code in this example be extended to do that for the user? The fewer steps the better!